R/prep_scatter.R

```diff
@@ -1,19 +1,20 @@
 #' Prepare data to plot scatterplot
 #'
 #' @param data_bopo data.frame. Data containing buildout and phaseout alignment
-#'   values. Must contain columns: 'group_id', 'year', 'sector', 'region',
-#'   'direction' and either 'name_abcd' and 'alignment_metric' or
-#'   'exposure_weighted_net_alignment'.
+#'   values. Must contain columns: `by_group`, `'year'`, `'sector'`, `'region'`,
+#'   `'direction'` and either `'name_abcd'` and `'alignment_metric'` or
+#'   `'exposure_weighted_net_alignment'`.
 #' @param data_net data.frame. Data containing net alignment values. Must
-#'   contain columns: 'group_id', 'year', 'sector', 'region', 'direction' and
-#'   either 'name_abcd' and 'alignment_metric' or
-#'   'exposure_weighted_net_alignment'.
-#' @param data_level Character. Level of the plotted data. Can be 'bank' or
-#'   'company'.
+#'   contain columns: `by_group`, `'year'`, `'sector'`, `'region'`,
+#'   `'direction'` and either `'name_abcd'` and `'alignment_metric'` or
+#'   `'exposure_weighted_net_alignment'`.
+#' @param data_level Character. Level of the plotted data. Can be `'bank'` or
+#'   `'company'`.
 #' @param year Integer. Year on which the data should be filtered.
 #' @param sector Character. Sector to filter data on.
 #' @param region Character. Region to filter data on.
-#' @param group_ids_to_plot Character vector. Bank ids to filter on.
+#' @param by_group Character. Vector of length 1. Variable to group by.
+#' @param groups_to_plot Character vector. Bank ids to filter on.
```

Comment:
Here 'Bank' should probably be changed to something else?

Reply:
indeed